### DISCUSSION TITLE
Lite Version

### DISCUSSION THREAD

**Discussion by MehdiGu** [Vedza/NitroSniperGo#155] [Ideas]
make or add a lite version like (snipe only or giveaway only or joining servers only ) so it can be faster cuz the delay is 250 ms and thats is to slow

**Comment by Vedza** [comment]
You can already disable giveaway, invite and privnote sniper in the settings. And maybe you internet speed is bad and/or the computer your sniper is running on is too far from the server hosting the guild. Here is the delays I have on heroku with everything enabled
![image](https://user-images.githubusercontent.com/28839427/106358676-ef924f80-630d-11eb-932e-fe715661960d.png)

**Reply by MehdiGu** [reply]
im runing it on a rdp and i compared it white StealthSniper and this is the result :
StealthSniper  
![stealth](https://user-images.githubusercontent.com/46828634/106358884-3af92d80-630f-11eb-8839-e4d505a192e1.png)

NitroSniperGo
![go](https://user-images.githubusercontent.com/46828634/106358913-5cf2b000-630f-11eb-91fe-c3b38b4f0df9.png)
white everything off

**Reply by Vedza** [reply]
As I said it's "hard to compare with other snipers for various reasons", and some of these reasons is that we don't print delay between the same steps and/or some are just fake.
